src/main/java/com/npsproject/it4045npsproject/controller/NPSController.java

```diff
@@ -21,14 +24,25 @@ public NPSController(ParkService parkService, ParkSearchService parkSearchServic
         this.parkSearchService = parkSearchService;
     }
 
-    @GetMapping("/{name}")
+    /**
+     * Return a park based on the name inputted by the end user
+     * @param name string entered by end user in the interface
+     * @return park and park details
+     */
+    @GetMapping("/park/{name}")
     public Park getParkByName(@PathVariable String name) {
         return parkService.getParkDetails(name);
     }
 
-    @PostMapping("/search")
-    public ParkSearchResult searchParks(@RequestBody ParkSearchRequest request) {
-        return parkSearchService.searchParks(request);
+    /**
+     * Return all parks
+     * @return list of all park objects
+     */
+    @GetMapping("/park")
```

Comment:
Updated name to use a noun instead of a verb.
https://blog.dreamfactory.com/best-practices-for-naming-rest-api-endpoints/